docs/reference/bundles/price.rst

```diff
@@ -18,23 +18,19 @@ Price bundle handles everything related to prices, (right now, only currencies,
 Installation
 ============
 
-This central e-commerce bundle requires you add a `bcscale <http://php.net/manual/en/function.bcscale.php>`_ method in your ``AppKernel.php`` file to ensure that prices are correctly computed.
-Please, update your ``AppKernel.php`` file like this:
+This central e-commerce bundle requires you add a `bcscale <http://php.net/manual/en/function.bcscale.php>`_ method in your ``Kernel.php`` file to ensure that prices are correctly computed.
+Please, update your ``Kernel.php`` file like this:
 
 .. code-block:: php
 
     <?php
 
-    // ...
+    ...
 
-    class AppKernel extends Kernel
+    public function registerBundles()
     {
-        public function init()
-        {
-            bcscale(3); // or any other value greater than 0
-
-            ...
-
+        bcscale(3);
```

Comment:
I think this function call would be better in the `boot` method of the kernel

```php
public function boot()
{
    bcscale(3); // or any other value greater than 0
    parent::boot();
}
```

Reply:
Yes seems better to me too, thanks!